server/src/server/Grakn.java

```diff
@@ -64,8 +64,13 @@ public static void main(String[] args) {
             boolean benchmark = parseBenchmarkArg(args);
             Server server = ServerFactory.createServer(benchmark);
             server.start();
-
             LOG.info("Grakn started in {}", timer.stop());
+            try {
+                server.awaitTermination();
+            } catch (InterruptedException e) {
+                // grakn server stop is called
+                server.close();
```

Comment:
I think we need to add `Thread.currentThread().interrupt()`, base on [SonarCloud rule](https://rules.sonarsource.com/java/tag/error-handling/RSPEC-2142)